climpred/metrics.py

```diff
@@ -1737,19 +1745,41 @@ def _brier_score(forecast, verif, dim=None, **metric_kwargs):
         Option 1. Pass with keyword `logical`: (Works also for PerfectModelEnsemble)
 
         >>> hindcast.verify(metric='brier_score', comparison='m2o',
-                dim='member', alignment='same_verifs', logical=pos)
```

Comment:
The only question I have about this PR: do you think starting the example from `hindcast` is enough @bradyrx? I skip the `PredictionEnsemble()` initialization part which would be 100% repetitive and just take it from the `conftest.py` internally.

Reply:
@aaronspring, I think it's fine to just have `hindcast.verify(...)` here. It would be really redundant to set it up every time and I think the user should see from other docstrings/examples how to set up the hindcast object. I think it's clear here what is being referenced.

Can `doctest` handle this passing even though there's no imports or anything to set up hindcast?

Reply:
Yes. See in conftest.py but the file had to be move to be above climpred and not only tests